### ISSUE TITLE
RSS block: how to have more than 10 items?

### ISSUE THREAD

**Issue by ghost** [WordPress/gutenberg#24859]
I can not have more than 10 items with RSS block.

Try to remove the limit of 10 items.

**Comment by aristath** [comment]
The `10` limit was part of the initial implementation in https://github.com/WordPress/gutenberg/pull/7966, but as far as I can see there was no discussion around the number, it was simply deemed reasonable - and in most cases I'd say it probably is.
I can see however why some would want a larger number... Perhaps we could make it easy to modify.

**Comment by andrewpalo** [comment]
Maybe to show jobs

**Comment by sabernhardt** [comment]
Does the `RangeControl` require a `max` value for the number field? If so, it probably could be as high as 100 (same range as the number of words in excerpts).

**As it is now:**
When typing a larger number in the input and clicking Publish (or Update) immediately after, I was able to _get away with_ exceeding the maximum. However, when I clicked on that block again, it corrected the number to the maximum value of 10.
